### DISCUSSION TITLE
Kanban done subcolumns and flow efficiency

### DISCUSSION THREAD

**Discussion by MichaelBar1975** [nbrown02/FlowViz#75] [Q&A] [answered]
About the 'flow efficiency' report , is it fully based on states or can it also take into account the done "sub" column that can be used in Azure DevOps Kanban boards?

![image](https://user-images.githubusercontent.com/43378596/162155166-0a1b5b70-3416-4217-b95e-b0822fbdbe72.png)

Or do we need to split up our Kanban columns and give the done columns a separate state?

**Comment by nbrown02** [comment, marked as answer]
Hey Michael,

Ah that's a great question. I hadn't considered that in my setup! So it won't do currently, however I can investigate to see if I can add that in :)

**Reply by nbrown02** [reply]
So I just had a quick look, I'm fairly confident this can be done as there is an 'IsDone' column in the back end for split columns. 

Will require a bit of a rejigging of formulas and slicer tables so a few hours work but confident I can get something done over the weekend...

**Comment by MichaelBar1975** [comment]
Wow that would be great!

Anyway flowviz is a really an amazing collection of reports and they are becoming more important in our team to see how we are doing. Thanks for this great tool.

**Reply by nbrown02** [reply]
Not a problem - I actually completely forgot about the sub columns when I did the Flow Efficiency page, it’s a really good idea so thank you 👍🏼

Glad you like it - there is an online training course also now available -  https://flowviz.thinkific.com/courses/fundamentals LAUNCH30 gets you 30% off 😊

**Comment by nbrown02** [comment]
@MichaelBarScia - just deployed a new version where I think I have sorted this for you :)

**Reply by MichaelBar1975** [reply]
Well I've tested it and looks it correct to me. Thanks! I'm happy I can finally show our 'flow efficiency' to the team (which isn't good but we are improving, so it will be nice to make it visible).

**Reply by nbrown02** [reply]
Ah fantastic! Thank you again for this idea, was a nice little challenge. If you have time, it would be great to leave a review on the Azure DevOps marketplace :)

**Reply by MichaelBar1975** [reply]
Done!
